### ISSUE TITLE
Weird error Undefined symbols for architecture x86_64

### ISSUE THREAD

**Issue by Otetz** [cosmo0920/tesseractocr-capi-go#3]
OS X:
```
$ go run cmd/phelper/main.go
# command-line-arguments
/usr/local/go/pkg/tool/darwin_amd64/link: running clang++ failed: exit status 1
Undefined symbols for architecture x86_64:
  "tesseract::TessBaseAPI::SetImage(Pix*)", referenced from:
      _simple in 000005.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

$ tesseract -v
tesseract 3.02.02
 leptonica-1.72
  libgif 4.2.3 : libjpeg 8d : libpng 1.6.28 : libtiff 4.0.7 : zlib 1.2.8 : libopenjp2 2.1.2
```

**Comment by cosmo0920** [comment]
Could you show minimal reproducible code and paste `otool -L /usr/local/bin/tesseract` result?

**Comment by Otetz** [comment]
```
$ otool -L /usr/local/bin/tesseract
/usr/local/bin/tesseract:
	/usr/local/lib/libtesseract.3.dylib (compatibility version 4.0.0, current version 4.2.0)
	/usr/local/lib/liblept.4.dylib (compatibility version 5.0.0, current version 5.3.0)
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.8)
	/usr/local/opt/libpng/lib/libpng16.16.dylib (compatibility version 45.0.0, current version 45.0.0)
	/usr/local/opt/jpeg/lib/libjpeg.8.dylib (compatibility version 13.0.0, current version 13.0.0)
	/usr/local/opt/giflib/lib/libgif.4.dylib (compatibility version 6.0.0, current version 6.7.0)
	/usr/local/opt/libtiff/lib/libtiff.5.dylib (compatibility version 8.0.0, current version 8.5.0)
	/usr/local/opt/openjpeg/lib/libopenjp2.7.dylib (compatibility version 7.0.0, current version 2.1.2)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 307.4.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1238.0.0)
```

**Comment by cosmo0920** [comment]
Could you paste the following command results?

```log
$ brew info tesseract
$ brew info cosmo0920/tesseract-versioned/tesseract302
```

**Comment by Otetz** [comment]
In brew not available `leptonica-1.72`, only current `leptonica-1.74`. In that case, I compile both from sources.

But I found what was a problem :) This was interference with gosseract, my mistake, sorry.

**Comment by cosmo0920** [comment]
Ah, I got it.
Thanks for Investigating the root of causing this problem. :)
